frontend/src/components/Category.jsx

```
@@ -0,0 +1,74 @@
+              className="w-full h-[120px] sm:h-[200px] object-cover transition-transform duration-300 group-hover:scale-105"
+            />
+            <div className="absolute inset-0 bg-black bg-opacity-0 group-hover:bg-opacity-40 transition-all duration-300 flex items-center justify-center p-2">
+              <div className="text-center text-white opacity-0 group-hover:opacity-100 transition-opacity duration-300">
+                <h3 className="text-2xl font-semibold sm:mb-2">
+                  {category.name}
+                </h3>
+                <p className="text-sm mb-2 sm:mb-4">{category.description}</p>
+                <button
+                  onClick={() => handleClick(category.name)}
+                  className="bg-white text-black px-4 py-2 rounded-full font-medium hover:bg-gray-200 transition-colors"
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)